src/plugins/dam/components/image-asset-picker-modal/index.jsx

```diff
@@ -29,6 +29,7 @@ class ImageAssetPickerModal extends React.Component {
     galleryNode: PropTypes.instanceOf(Message),
     isOpen: PropTypes.bool,
     label: PropTypes.string,
```

Comment:
@bhsiao55 i don't think you need the label props now really! you can just default your 'title' to current label's value.

Reply:
label still being used by other components